Blog/views.py

```diff
@@ -146,6 +147,12 @@ class PostCreate(LoginRequiredMixin, generic.CreateView):
     def form_valid(self, form):
         form.instance.user = self.request.user
         form.instance.created = datetime.datetime.now()
+        slug = slugify(form.instance.title)
+        same_slugs_count = Post.objects.filter(slug__icontains=form.instance.slug).count()
```

Comment:
Ooops, I nearly missed this: you should use `slug`to do the lookup because `form.instance.slug` is an empty string at this point (it's only set later).